modules/workstation/homed.nix

```
@@ -0,0 +1,5 @@
+{ lib, config, ... }: {
+  config = lib.mkIf config.modules.desktop.enable {
```

This module is in `modules/workstation`, so it should be activated by `config.modules.workstation.enable`

```suggestion
  config = lib.mkIf config.modules.workstation.enable {
```